policyengine_canada/parameters/gov/provinces/mb/credits/basic_personal_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Basic Personal Amount in Manitoba. 
+values:
+  2023-01-01: 15_000
```

Comment:
please add the 2022 value as well https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1mb/td1mb-22e.pdf